scripts/projectUpdater.ps1

```diff
@@ -117,6 +117,26 @@ $_templateMetadata =
     $_metadata.Templates |
         Where-Object { $_.folder -eq $templateName }
 
+# If the template is deprecated or broken, it cannot be updated. If it is incomplete the user should be made aware of the problem and choose to proceed or not.
+if ($_templateMetadata.status -eq "deprecated") {
+    Write-Host -ForegroundColor DarkRed "This template is deprecated in the most recent version of the Torizon IDE Extension. For details, check https://github.com/torizon/vscode-torizon-templates/blob/dev/DEPRECATED.md"
+    exit 0
+
+} elseif ($_templateMetadata.status -eq "notok") {
+    Write-Host -ForegroundColor DarkRed "This template is broken in the most recent version of the Torizon IDE Extension. Reason:"
+    Write-Host -ForegroundColor DarkRed $_templateMetadata.customMessage
+    exit 0
+
+} elseif ($_templateMetadata.status -eq "incomplete") {
```

Comment:
Also would be nice to have on the documentation, here and developer, what means the "deprecated"/"notok"/"imcomplete" and the implications.

Reply:
Sure, will add it to the README of the templates

Reply:
Added a section at: https://github.com/torizon/vscode-torizon-templates/pull/288/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R57